### PR TITLE
ARUHA-648: extended logging

### PR DESCRIPTION
**Pull request by v-stepanov**
Just to see for whom it happens next time.

### CODE REVIEW

**Comment by lmontrieux** [comment]
:shipit:

File: src/main/java/org/zalando/nakadi/service/EventStream.java [hunk 1]
```diff
@@ -122,7 +122,8 @@ public void streamEvents(final AtomicBoolean connectionReady) {
         } catch (final IllegalStateException e) {
             LOG.info("Error occurred when streaming events (possibly server closed connection)", e);
         } catch (final KafkaException e) {
-            LOG.error("Error occurred when polling events from kafka", e);
+            LOG.error("Error occurred when polling events from kafka; consumer: " + config.getConsumingAppId() +
```
Comment (adyach) [inline src/main/java/org/zalando/nakadi/service/EventStream.java]: We usually use `{}` for params

Comment (v-stepanov) [inline src/main/java/org/zalando/nakadi/service/EventStream.java]: But then I will not be able to provide exception in the log.

Comment (adyach) [inline src/main/java/org/zalando/nakadi/service/EventStream.java]: you can pass it as the latest param
https://www.slf4j.org/faq.html#paramException

Comment (antban) [inline src/main/java/org/zalando/nakadi/service/EventStream.java]: @v-stepanov I agree with @adyach

**Comment by adyach** [comment]
👍

File: src/main/java/org/zalando/nakadi/service/EventStream.java (hunk 1 above)
Comment (v-stepanov) [inline src/main/java/org/zalando/nakadi/service/EventStream.java]: @adyach cool, I didn't know about that. I changed it now.

**Comment by adyach** [comment]
:shipit:

**Pull request merged**